Project.toml

```diff
@@ -21,7 +21,7 @@ FFTW = "1"
 JOLI = "0.7"
 PyCall = "1.18, 1.90, 1.91, 1.62"
 Reexport = "0.2, 1"
-SegyIO = "0.7"
+SegyIO = "0.7.6"
```

Comment:
nerd 0.7.7 for some reason 0.7.6 already existed not sure why wasn't in the project file